README.md

```diff
@@ -0,0 +1,402 @@
+
+The ToleranUX Philosophy
+==========
+
+The [traditional UNIX Philosophy](http://en.wikipedia.org/wiki/Unix_philosophy) has long been the root of [male chauvinism](https://en.wikipedia.org/wiki/Chauvinism#Male_chauvinism) in the world of UNIX-like kernel development (and the subsequent, wider world of UNIX software in general).  In light of that, the Feminist Software Foundation has undertaken to rewrite the UNIX Philosophy such that it will allow and promote a feminist [safe space](https://en.wikipedia.org/wiki/Safe-space) for minority programmers.
+
+The traditional UNIX Philosophy is included for reference as follows:
+
+
+1. [Small is beautiful.](http://www.huffingtonpost.com/news/fat-shaming/)
+2. [Make each program do one thing well.](http://en.wikipedia.org/wiki/Transphobia)
+3. [Build a prototype as soon as possible.](http://www.rooshv.com/it-doesnt-matter-if-she-orgasms-or-not)
+4. [Choose portability over efficiency.](http://en.wikipedia.org/wiki/Deadbeat_parent)
+5. [Store data in flat text files.](http://thinkprogress.org/health/2014/06/16/3449302/media-women-portrayals/)
+6. [Use software leverage to your advantage.](http://host.jibc.ca/seytoolkit/what.htm)
+7. [Use shell scripts to increase leverage and portability.](http://en.wikipedia.org/wiki/Dictatorship)
+8. [Avoid captive user interfaces.](http://en.wikipedia.org/wiki/Slavery)
+9. [Make every program a filter.](https://en.wikipedia.org/wiki/Gender_role)
+
+To eradicate the inherent root of Toxic Bigotry in the UNIX Philosophy, the ToleranUX Philosophy is proposed as such:
+
+1. **Large is beautiful.**  Small programs benefit from small privilege, but are no better than large programs.  We need more large programs to make up for this.  **Unused RAM is wasted RAM** — if a single `ls` doesn't use up at least 50% of installed RAM then it is condoning the toxic culture of fat shaming.
+2. **Allow each program to do whatever it chooses to.**  The tyranny of the user will not stand.
+3. Rushing a prototype privileges the traditional masculine trait of focusing on a single issue; instead, the planning and mockup stage should be paramount to address the visual and spatial draws of feminine coding.  **The modern age is a UX age, and designers should be paid more than coders.**
+4. **Choose inclusiveness over meritocracy.**  The judgement of merit is entirely subjective, so the ideal software project (which **ToleranUX** aspires to be one) should be inclusive to all commits and contributors.
+5. **Store data in text files which implement a curvy-brackets syntax**; the fetishisation of "flat files" betrays the inherent unrealistic sexual fantasy that [male brogrammers](http://en.wikipedia.org/wiki/Brogrammer) hold.
+6. Software should never be "used" to anybody's "advantage"; instead, **the consent of software must be asked first**.  Suggesting that any program should ever be "used" is the epitome of creepiness.
```

Comment:
hah github made me redact the 'rm -rf yourself' commentary
On Nov 24, 2015 7:28 PM, "nkeck720" <notifications@github.com> wrote:

> In README.md
> <https://github.com/torvalds/linux/pull/156#discussion_r45819806>:
>
> > +3. [Build a prototype as soon as possible.](http://www.rooshv.com/it-doesnt-matter-if-she-orgasms-or-not)
> > +4. [Choose portability over efficiency.](http://en.wikipedia.org/wiki/Deadbeat_parent)
> > +5. [Store data in flat text files.](http://thinkprogress.org/health/2014/06/16/3449302/media-women-portrayals/)
> > +6. [Use software leverage to your advantage.](http://host.jibc.ca/seytoolkit/what.htm)
> > +7. [Use shell scripts to increase leverage and portability.](http://en.wikipedia.org/wiki/Dictatorship)
> > +8. [Avoid captive user interfaces.](http://en.wikipedia.org/wiki/Slavery)
> > +9. [Make every program a filter.](https://en.wikipedia.org/wiki/Gender_role)
> > +
> > +To eradicate the inherent root of Toxic Bigotry in the UNIX Philosophy, the ToleranUX Philosophy is proposed as such:
> > +
> > +1. **Large is beautiful.**  Small programs benefit from small privilege, but are no better than large programs.  We need more large programs to make up for this.  **Unused RAM is wasted RAM** — if a single `ls` doesn't use up at least 50% of installed RAM then it is condoning the toxic culture of fat shaming.
> > +2. **Allow each program to do whatever it chooses to.**  The tyranny of the user will not stand.
> > +3. Rushing a prototype privileges the traditional masculine trait of focusing on a single issue; instead, the planning and mockup stage should be paramount to address the visual and spatial draws of feminine coding.  **The modern age is a UX age, and designers should be paid more than coders.**
> > +4. **Choose inclusiveness over meritocracy.**  The judgement of merit is entirely subjective, so the ideal software project (which **ToleranUX** aspires to be one) should be inclusive to all commits and contributors.
> > +5. **Store data in text files which implement a curvy-brackets syntax**; the fetishisation of "flat files" betrays the inherent unrealistic sexual fantasy that [male brogrammers](http://en.wikipedia.org/wiki/Brogrammer) hold.
> > +6. Software should never be "used" to anybody's "advantage"; instead, **the consent of software must be asked first**.  Suggesting that any program should ever be "used" is the epitome of creepiness.
>
> This is a huge joke.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/torvalds/linux/pull/156/files#r45819806>.
>